lectures/learning1.js

```diff
@@ -709,8 +709,8 @@ add(summarySlide('Summary so far',
 _));
 
 prose(
-  'In summary, we have seen linear predictors, the functions we\'re considering',
-  'the criterion for choosing one, and an algorithm that goes after that criterion.',
+  'In summary we have seen linear predictors, the functions we\'re considering', // "the functions we're considering the criterion for choosing one"
+  'the criterion for choosing one, and an algorithm that goes after that criterion.', // Might be worth a rewording, but happy to keep as-is.
```

Comment:
maybe add (i) the functions we're considering (linear predictors), (ii) the criterion ... (loss minimization), and (iii) an algorithm ... (SGD)